model-setup-examples.md

```diff
@@ -0,0 +1,28 @@
+# Model Setup Examples
+
+We have put together some simple examples of how to set up the radio, depending on your needs.
+
+Some of the features we cover are:
+
+* Mixers
+* Special functions
+* Logic Switches
+* Flight modes
+
+OpenTx allows all sorts of interesting ways to program your transmitter.
+
+Hopefully these examples will get you started in the right direction.
+
+All examples are assuming a Taranis X9D+ and using the internal FrSky module in Mode 2.
+
+Because of this, the channel order in the examples with be AETR
```

Comment:
I can take the Mode 2 reference out...guess I thought I was going to talk about the left stick and the right stick...but never did.